### ISSUE TITLE
is not work from namespace

### ISSUE THREAD

**Issue by zhuamaodeyu** [realm/jazzy#1130]
1.Question 
if i set namespace for system class. Example:  

```
public final class XXXXExtension<Base> {
    public let base: Base
    public init(_ base: Base) {
        self.base = base
    }
}

public protocol XXXCompatible {
    associatedtype AKCompatibleType
    var ak: AKCompatibleType { get }
    static var ak: AKCompatibleType.Type { get }
}

public extension AXXXCompatible {
    
    public var ak: AXXXXExtension<Self> {
        return AXXXXExtension(self)
    }
    public static var ak: AXXXXExtension<Self>.Type {
        return AXXXXExtension.self
    }
}
```
 set document . Example: 

```
///this is  document 
extension AXXXXExtension where Base: NSObject {
    
    /// 
    public static var nameOfClass: String {
        return ""
    }
}

```
is not work .

**Comment by johnfairh** [comment]
Hello -- this code doesn't build and I'm not sure what you're pointing to as not working?

I can see there's a problem distinguishing the default implementations of the two `ak` fields -- is that it?

**Comment by zhuamaodeyu** [comment]
hi, you can find examples in this repository. ![TKUIKitModule](https://github.com/zhuamaodeyu/TKUIKitModule.git).   Example: the repository has a `Namespace.swift` file.  now,  I want to add an extension to UIApplication。 for this. [UIApplication+Ext.swift](https://github.com/zhuamaodeyu/TKUIKitModule/blob/master/TKUIKitModule/Classes/Extension/UIApplication%2BExt.swift). Ideal situation，The document should be generated under UIApplication。 but is now for NameSpace  . How do I need to configure it?

**Comment by johnfairh** [comment]
OK I see.  Jazzy can only document the code that is written.  The module has no `UIApplication` extension, just conditional extensions of `TypeWrapperProtocol`.  Right now, jazzy doesn't display conditional extensions very well at all -- by coincidence I'm working on improvements for this that will make that page look something like:

<img width="606" alt="Screenshot 2019-11-12 at 09 29 09" src="https://user-images.githubusercontent.com/26768470/68659303-e9f4f200-052e-11ea-8a1c-63e82e9c8179.png">

...which at least will get `UIApplication` etc in there.

**Comment by zhuamaodeyu** [comment]
Thanks for your reply, very looking forward to the new version

**Comment by jpsim** [comment]
0.12 was released a few days ago: https://github.com/realm/jazzy/releases/tag/v0.12.0

**Comment by johnfairh** [comment]
Fixed in master via #1131
